lib/util/package-rules/index.ts

```diff
@@ -4,22 +4,16 @@ import { mergeChildConfig } from '../../config';
 import type { PackageRule, PackageRuleInputConfig } from '../../config/types';
 import { logger } from '../../logger';
 import matchers from './matchers';
-import { matcherOR } from './utils';
 
 function matchesRule(
   inputConfig: PackageRuleInputConfig,
   packageRule: PackageRule,
 ): boolean {
-  let positiveMatch = true;
+  const positiveMatch = true;
```

Comment:
If this is now a const than it is no longer needed

Reply:
Ha, VSCode/eslint autochanged that, I didn't notice